### PR TITLE
Update session model

### PR DESCRIPTION
**Pull request by jiachen247**
This PR makes changes to the session model in line with PR #170

### CODE REVIEW

File: src/main/java/seedu/zerotoone/MainApp.java [hunk 1]
```diff
@@ -158,32 +158,34 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
...
+            logListOptional = storage.readLogList();
+            if (logListOptional.isEmpty()) {
                 logger.info("Data file not found. Will be starting with a sample ExerciseList");
             }
-            initialSessionListData = sessionListOptional.orElseGet(SampleSessionDataUtil::getSampleSessionList);
+            initialLogListData = logListOptional.orElseGet(SampleLogDataUtil::getSampleLogList);
 
             try {
-                storage.saveSessionList(initialSessionListData);
+                storage.saveLogList(initialLogListData);
```
Comment (Guofeng-Tang) [inline src/main/java/seedu/zerotoone/MainApp.java]: Jia chen you wanna move your saving code to execute method in LogicManager? We are all doing it there

**Review by Guofeng-Tang — DISMISSED** [review]

File: src/main/java/seedu/zerotoone/MainApp.java (hunk 1 above)
Comment (Guofeng-Tang) [reply src/main/java/seedu/zerotoone/MainApp.java]: btw sorry I think I broke your code when I tried to resolve the merge conflicts, wrong import coz of the name change of session, can just revert that please

Comment (jiachen247) [reply src/main/java/seedu/zerotoone/MainApp.java]: I have settled the merge conflict! No worries bro! And good catch have moved the save logic in to the logic manager! Thanks!

**Review by gb3h — APPROVED** [review]
LGTM

File: src/main/java/seedu/zerotoone/MainApp.java (hunk 1 above)
Comment (gb3h) [reply src/main/java/seedu/zerotoone/MainApp.java]: Verify that saving is done in logic manager so closed

**Pull request merged**